slurm2sql.py

```diff
@@ -59,7 +59,9 @@ def datetime_timestamp(dt):
 def slurmtime(x):
     """Parse slurm time of format [dd-[hh:]]mm:ss"""
     if not x: return None
-    if x == "Partition_Limit": return None
+    # Handle 'UNLIMITED' ,'Partition_Limit' in 'timelimit' field
+    if  x in ['Partition_Limit', 'UNLIMITED']:
```

Comment:
```suggestion
    if  x in {'Partition_Limit', 'UNLIMITED'}:
```
I did a quick test on python3.6 with timeit, for a string that wasn't in the list, and it seems that set was faster by a little bit. I haven't studied much but it should get more so the longer it is...